examples/ftp-service-receive-file/ftp_service_receive_file.metatags

```diff
@@ -1,2 +1,2 @@
-description: This BBE is about receiving file/directory changes that occur in a remote FTP server. It has a listener-based functionality for receiving these events.
+description: This example demonstrates receiving file/directory changes that occur in a remote FTP server. It has a listener-based functionality for receiving these events.
```

Comment:
This example is on listening to file changes and reading the changes from the caller right? IMO the existing description is confusing

Reply:
Fixed